pkg/manifests/manifests.go

```diff
@@ -1710,6 +1710,12 @@ func (f *Factory) PrometheusK8s(grpcTLS *v1.Secret, trustedCABundleCM *v1.Config
 
 	for i, container := range p.Spec.Containers {
 		switch container.Name {
+		case "prometheus":
+			// TODO (JoaoBraveCoding): Once prometheus-operator adds CRD support to configure startupProbe directly
+			// we should use that instead of using strategic merge patch
+			p.Spec.Containers[i].StartupProbe = &v1.Probe{
+				PeriodSeconds: 30,
```

Comment:
I'd rather keep the PeriodSeconds value unchanged and increase FailureThreshold. And for "safety", I'd set the timeout duration to 1h in total.

```suggestion
				PeriodSeconds: 15,
				FailureThreshold: 240,
```